### PR TITLE
Amendments to man page

### PR DESCRIPTION
**Pull request by digitensions**
Three commits and the man page is completed. I understand from @retokromer that this was taken from the help page, so will make amendments to help page fast.  I've noticed some failed status for the third commit, something to do with Travis.ci - any idea what that is? Will it impact the PR process?  Thanks and look forward to feedback. Joanna

### CODE REVIEW

File: Source/CLI/rawcooked.1 [english edits] [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+.br
+This is default.
+.TP
+.B \-\-no\-encode
+Do not carry out encode (see above).
+.TP
+.B \-\-hash
+Can be used to either compute, or test, the hash of files.
+.br
+To compute a hash use with the audio-visual RAW data. A hash is generated and stored in the RAWcooked metadata.
```
Comment (retokromer) [inline Source/CLI/rawcooked.1 [english edits]]: use `audio\-visual`

File: Source/CLI/rawcooked.1 [english edits] [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+.br
+The default value is \fI0\fR.
+.TP
+.B \-format \fIvalue
+Set the container format to \fIvalue\fR: only \fIMatroska\fR is currently allowed.
+.br
+The default value is \fIMatroska\fR.
+.TP
+.B \-g \fIvalue
+If video encoding format is \fIFFV1\fR, set the GOP size to \fIvalue\fR\: \fI1\fR (generates a strict intra-frame bitstream), \fI0\fR (allows adaptable context model across frames).
```
Comment (retokromer) [inline Source/CLI/rawcooked.1 [english edits]]: use `intra\-frames`

File: Source/CLI/rawcooked.1 [english edits] [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+If video encoding format is \fIFFV1\fR, you can set the CRC checksum to \fIvalue\fR: \fI0\fR (CRC checksums off), \fI1\fR (CRC checksum on).
+.br
+The default value is \fI1\fR.
+.TP
+.B \-slices \fIvalue
+If the video encoding format is \fIFFV1\fR, you can set the multithreaded encoding slices to \fIvalue\fR: any integer over 1 (it is recommended to use a figure divisible by your workstations CPU core processors such as 2, 4, 6, 9, 16, 24…).
+.br
+The default value is variable between \fI16\fR and \fI512\fR, depending on the video frame size and depth.
+.SH COPYRIGHT
+Copyright (c) 2019 MediaArea.net SARL & AV Preservation by reto.ch
```
Comment (retokromer) [inline Source/CLI/rawcooked.1 [english edits]]: Why is 2018 deleted?

File: Source/CLI/rawcooked.1 [english edits] [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+.br
+The default value is the latest version \fI3\fR.
+.TP
+.B \-slicecrc \fIvalue
+If video encoding format is \fIFFV1\fR, you can set the CRC checksum to \fIvalue\fR: \fI0\fR (CRC checksums off), \fI1\fR (CRC checksum on).
+.br
+The default value is \fI1\fR.
+.TP
+.B \-slices \fIvalue
+If the video encoding format is \fIFFV1\fR, you can set the multithreaded encoding slices to \fIvalue\fR: any integer over 1 (it is recommended to use a figure divisible by your workstations CPU core processors such as 2, 4, 6, 9, 16, 24…).
```
Comment (retokromer) [inline Source/CLI/rawcooked.1 [english edits]]: use three periods `...` rather that the typographic `…` (which gives `a|` in man)

**Comment by digitensions** [comment]
Thanks @retokromer that explains my fail then.  @JeromeMartinez has indicated that he will complete these alterations for me, and I'll move onto the Docs folder changes. Many thanks both!  Great fun to be editing and running PRs one after another. A great way to learn, so thanks for thinking of me!

File: Source/CLI/rawcooked.1 [english edits] (hunk 4 above)
Comment (digitensions) [reply Source/CLI/rawcooked.1 [english edits]]: Ah, thanks for sharing this! How did you run it visually like that to test it?

File: Source/CLI/rawcooked.1 [english edits] (hunk 3 above)
Comment (digitensions) [reply Source/CLI/rawcooked.1 [english edits]]: Whoops. Thought it was out of date! Ignore me :)

File: Source/CLI/rawcooked.1 [english edits] [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+.if n .ad l
+.\" Turn off hyphenation.
+.nh
+.SH NAME
+\fBRAWcooked\fR \- encode and decode audio\-visual RAW data with Matroska, FFV1 and FLAC
+.SH SYNOPSIS
+\fBrawcooked \fR[\fIoption\fR ...] (\fIfolder\fR | \fIfile\fR ...) [\fIoption\fR ...]
+.SH DESCRIPTION
+.TP
+\fBRAWcooked\fR easily encodes RAW audio\-visual sequences into a lossless video stream, reducing the file size by between one and two thirds. FFmpeg encodes the audio\-visual data into a Matroska container (MKV) using the video codec FFV1, and audio codec FLAC. The metadata accompanying the RAW data is fully preserved, along with additional sidecar files such as MD5 checksums, LUT or XML if desired. This allows for the management of these audio-visual file formats in an effective and transparent way. The lossless Matroska video stream can be played in VLC or MPV media players, and writing and retrieving from storage devices such as LTO is significantly quicker.
```
[the remark this answers is not in the document]
Comment (digitensions) [reply Source/CLI/rawcooked.1 [english edits]]: Sorry for all the missed \-

File: Source/CLI/rawcooked.1 [english edits] (hunk 4 above)
Comment (retokromer) [reply Source/CLI/rawcooked.1 [english edits]]: Just tape into the terminal: 
man <drag&drop_your_ammended_local_file_here>

File: Source/CLI/rawcooked.1 [english edits] (hunk 5 above)
[the remark this answers is not in the document]
Comment (JeromeMartinez) [reply Source/CLI/rawcooked.1 [english edits]]: https://github.com/MediaArea/RAWcooked/pull/227 removed `\\` before `-`, so is it relevant here to add `\\` before `-`?

**Comment by JeromeMartinez** [comment]
I rebased the PR so it is including all new commits merged in master branch since the last update of this PR.
I reverted the file renaming (there was an unwanted "rawcooked.1 [english edits]" file renaming in the process).

Please review changes.

**Comment by JeromeMartinez** [comment]
@retokromer I think I edited as you expect (`Matroska`/`FFV1` for the name, `matroska`/`ffv1` for the command values), and fixed other issues, please review again.

File: Source/CLI/rawcooked.1 [hunk 6]
```diff
@@ -9,46 +9,46 @@
...
 .TP
-When needed, the uncompressed source is retrieved \fBbit-by-bit\fR, in a manner faster than uncompressed sources. 
+If you need to use the RAW source in its original form, one line of code will easily restore it \fBbit-by-bit\fR, faster than retrieving the same file from LTO tape storage.
 .TP
 .I folder
-Encodes with the FFV1 video codec all single-image video files in the \fIfolder\fR, encodes with the FLAC audio codec all audio files in the \fIfolder\fR, and muxes these into a Matroska container (.mkv).
+Every image file within the sequence's \fIfolder\fR is encoded into a single FFV1 video stream, and each audio track within the same \fIfolder\fR is encoded to the FLAC codec. Both the FFV1 and FLAC contents are then muxed into a single Matroska container (.mkv).
 .br
-The filenames of the single image files must end with a numbered sequence. \fBRAWcooked\fR will generate the regex to parse in the correct order all the frames in the folder.
+The image filesnames must end in a complete number sequence, so that \fBRAWcooked\fR can parse each image within the sequence in the correct order.
```
Comment (digitensions) [inline Source/CLI/rawcooked.1]: filenames / not file**s**names?

**Pull request merged**